### PR TITLE
changing load_balancer.md

### PR DESCRIPTION
**Pull request by enraiha0307**
Hi @CathPag, I made some changes. Could you please review it?

### CODE REVIEW

File: content/en/load_balancer.md [hunk 1]
```diff
@@ -3,12 +3,15 @@ title: Load Balancer
 status: Feedback Appreciated
 category: concept
 ---
-
 ## What it is
-A load balancer is a method to distribute incoming network traffic across a group of servers in the back-end. The solution can be software or hardware based.
+
```
Comment (jihoon-seo) [inline content/en/load_balancer.md:7]: For anyone's information:
it is good to insert empty lines before and after the headers, like this. 😊

- https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines
- https://arcticicestudio.github.io/styleguide-markdown/rules/headings.html#empty-lines-before-and-after

File: content/en/load_balancer.md [hunk 2]
```diff
@@ -3,12 +3,15 @@ title: Load Balancer
 status: Feedback Appreciated
```
Comment (jihoon-seo) [inline content/en/load_balancer.md:3]: When this PR is okay to be merged, please change the `status` to `Completed`:

```suggestion
status: Completed
```

**Review by jihoon-seo — COMMENTED** [review]
@enraiha0307 Thank you for your contribution 😊

File: content/en/load_balancer.md (hunk 1 above)
Comment (enraiha0307) [reply content/en/load_balancer.md:7]: Yes, I'll do that. Thanks

File: content/en/load_balancer.md [hunk 3]
```diff
@@ -5,10 +5,14 @@ category: concept
 ---
 
 ## What it is
-A load balancer is a method to distribute incoming network traffic across a group of servers in the back-end. The solution can be software or hardware based.
+
+A load balancer is a way to distribute incoming user requests among multiple instances of an application. The solution can be software or hardware based.
```
Comment (CathPag) [inline content/en/load_balancer.md:9]:
```suggestion
A load balancer is a tool that distributes incoming requests among multiple instances of an application. Each service of a [microservices](/microservices/) app generally has multiple replicas (or instances), and the load balancer ensures no one service gets the bulk of the requests. Load balancers can be software or hardware-based.
```

File: content/en/load_balancer.md [hunk 4]
```diff
@@ -5,10 +5,14 @@ category: concept
...
 
 ## Problem it addresses
-This helps solve the issue related to high availability and distributed systems. When working on an application or service that needs to scale to hundreds of thousands of users, one will often need to distribute that application on multiple servers. The load balancer is the "traffic cop" that routes traffic.
+
+Suppose a person starts a business and builds a website. Initially, the number of people visiting their site would be less hence the speed with which the server(a computer that sends some information in response to a user’s request) serves the website would be optimal. Now the person invests in some marketing and their business starts to grow. This would result in an increased number of users visiting their website (probably from all around the globe). One server might not be enough to handle the increased number of requests. And would result in a slower website and frequent crashing of the server. One would have to add more servers to handle these many requests. But what if one of the servers goes down, what happens to the users it was serving? There was no way of keeping a check on these situations.
```
Comment (CathPag) [inline content/en/load_balancer.md:13]: This example is making a case for replicas, not a load balancer. Without a load balancer, all requests might be routed to the same service while other services are available. 

Also, please keep the examples short and sweet. A person building a website...people visiting the website...investing in marketing...business grows... that are too many details that are quite distracting.

**Review by CathPag — COMMENTED** [review]
Thanks for your contribution! 
I made a suggestion for the first section. The 2nd needs adjustments and the third should build on those.

File: content/en/load_balancer.md (hunk 4 above)
Comment (enraiha0307) [reply content/en/load_balancer.md:13]: Yes I'll make the changes and update.

**Comment by enraiha0307** [comment]
Hi @CathPag, Was busy with office work. I'll update it today :)

**Comment by enraiha0307** [comment]
Hi, @CathPag I have updated it, can you please review :)

File: content/en/load_balancer.md (hunk 3 above)
Comment (CathPag) [reply content/en/load_balancer.md:9]: What about this suggestion? Can we commit?

File: content/en/load_balancer.md (hunk 1 above)
Comment (CathPag) [reply content/en/load_balancer.md:8]: If this is resolved, please mark it as such.

**Review by CathPag — COMMENTED** [review]
@enraiha0307, could you please make sure all comments are resolved before requesting a new review? That makes things for us a lot easier :)

File: content/en/load_balancer.md (hunk 2 above)
Comment (enraiha0307) [reply content/en/load_balancer.md]: Does this term need to be changed by me or by the maintainers?

**Comment by enraiha0307** [comment]
Hi @CathPag, I have incorporated your suggestions, and marked the review comments as resolved.
One thing I want to ask is, The status has to be changed after this pull request gets approved by me or by the maintainers?
> Reminder for all: if this term gets okay to be displayed on the CNCF Glossary website, please change the `status` to `Completed`:

File: content/en/load_balancer.md [hunk 5]
```diff
@@ -5,10 +5,13 @@ category: concept
...
+
+In a scenario where multiple application instances are running on a group of servers, It can become tedious to manage and direct millions of incoming user requests toward available servers. A server might become overloaded with requests, resulting in reduced throughput, slower response time, and inefficient utilization of resources.
 
 ## How it helps
-A load balancer will act as front-end for network traffic and clients. It often has various methods to check which server(s) is up and has the lowest load to handle the request.
\ No newline at end of file
+
+Adding a load balancer will help in distributing the requests among the multiple servers in a way that would maximize the website's speed and ensure that no server is overworked. If any of the servers goes down, the load balancer will redirect the requests to other working servers. Also in case of [horizontal scaling](/horizontal_scaling/), it would automatically start sending the requests to the new server.
```
[the remark this answers is not in the document]
Comment (JasonMorgan) [reply content/en/load_balancer.md:17]: Just to add on to what Catherine said, Load balancing is a crucial technology that underpins cloud native software. Both IaaS hosted load balancers, like ALBs/NLBs, and in platform load balancing, Kubernetes services or a service mesh, are examples of cloud native load balancing. Nearly every app running in a microservice platform interacts with load balancers in one form or anther.

**Comment by CathPag** [comment]
Since I haven't heard back, I'll unassign this from you @enraiha0307

**Pull request closed without merging**